packages/react/src/components/Popover/Popover.mdx

```diff
@@ -9,32 +9,81 @@ import * as PopoverStories from './Popover.stories.tsx';
 <Primary />
 <Controls />
 
+## Slik bruker du `Popover`
+
+```tsx
+import '@digdir/design-system-tokens/brand/altinn/tokens.css'; // Importeres kun en gang i appen din.
+import { Popover } from '@digdir/design-system-react';
+
+<Popover>
+  <Popover.Trigger>Trigger</Popover.Trigger>
+  <Popover.Content>Content</Popover.Content>
+</Popover.Trigger>
+```
+
 `Popover.Trigger` styrer om innholdet i `Popover` skal vises eller ikke. Du kan bruke `asChild` for å endre hvordan `Popover.Trigger` rendres.
+Du kan bruke `anchorEl`, men denne vil bli fjernet i fremtiden, og vi anbefaler å bruke `Popover.Trigger` istedenfor. Du kan lese mer om `anchorEl` [her](#anchorel).
+
+`Popover.Content` er innholdet som skal vises i `Popover`.
+
```

Comment:
This is kind of redundant (and reads kinda wierd with the next chapter) as we have already mentioned this in the `Popover.Trigger` chapter further down.
```suggestion
```